caddyhttp/fastcgi/fastcgi.go

```diff
@@ -35,6 +41,9 @@ type Handler struct {
 func (h Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) (int, error) {
 	for _, rule := range h.Rules {
 
+		//TODO: add an option in Caddyfile and pass it down to here
+		usePersistentFcgiConnections := runtime.GOOS == "windows"
```

Comment:
I have tested a little ajax load using 
https://github.com/Echsecutor/fastcgi-test
on a ubuntu Laptop. Result: 

usePersistentFcgiConnections := true
the time for 101 requests varies between 0.7s and 1.6s

usePersistentFcgiConnections := false
the time for 101 requests varies between 0.6s and 1.5s

In other words, this measurements where not conclusive. ;) Not keeping the connection open and not serialising requests feels a little bit faster on Linux, but this might be a misconception.